app/models/search_query.rb

```diff
@@ -0,0 +1,9 @@
+class SearchQuery < ActiveRecord::Base
+	belongs_to :user
+
+	validates :user, presence: true
+
+	def search_query
+		"#{self.first_name} #{self.last_name} #{self.address} #{self.zipcode} #{self.phone_number}"
```

Comment:
What's with the weird indentation here? Make sure everyone has Sublime set to use spaces instead of tabs and 2 spaces for a tab.